src/log_windows.cpp

```diff
@@ -86,7 +86,7 @@ log_file_manager::log_file_manager(bool native_console)
 		// with the console subsystem flag and that the standard streams are
 		// already pointing to the console.
 		LOG_LS << "Console already attached at startup (built with console subsystem flag?), log file disabled.";
-	} else if(native_console) {
+	} else {
```

Comment:
Does this run even when logging to a file? That seems wrong. Or am I misinterpreting something?

Reply:
This function only gets called if logging to file has been disabled.

Reply:
That seems to imply that it is misnamed though.

Reply:
The `log_file_manager` class?

Reply:
Yes, if it's meant for logging to a console, not a file, then why is it called `log_file_manager`?

Reply:
Updated.